src/components/ActivityFeed/TabNavigation.tsx

```
@@ -0,0 +1,383 @@
+													</span>
+												</span>
+											</p>
+										)
+									);
+								})}
+							</div>
+						}
+						placement={'right'}
+						trigger='hover'
+						arrow={true}
+					>
```

Biome

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)